src/tike/ptycho/ptycho.py

```diff
@@ -494,19 +504,21 @@ def iterate(self, num_iter: int) -> None:
                     self.parameters.probe,
                     self.parameters.object_options.preconditioner,
                 )))
+            # ELSE IF: self.parameters.probe_options.rescale_using_fixed_probe_intensity_photons
+            # ELSE:    do nothing
+ 
+            if self.parameters.probe_options.recover_probe :
```

Comment:
You're using recover_probe here, but update_probe in the options class definition.

Reply:
fixed